assets/css/stripeStyle.css

```diff
@@ -0,0 +1,110 @@
+* {
+    box-sizing: border-box;
+}
+  
+body,
+html {
+    background-color: #f6f9fc;
+    font-size: 13px;
+    font-family: Helvetica Neue, Helvetica, Arial, sans-serif;
+    margin: 0;
+}
+
+label {
+    color: #6b7c93;
+    font-weight: 300;
+    letter-spacing: 0.025em;
+}
+
+button {
+    white-space: nowrap;
+    border: 0;
+    outline: 0;
+    display: inline-block;
+    height: 40px;
+    line-height: 40px;
+    padding: 0 14px;
+    box-shadow: 0 4px 6px rgba(50, 50, 93, 0.11), 0 1px 3px rgba(0, 0, 0, 0.08);
+    color: #fff;
+    border-radius: 4px;
+    font-size: 15px;
+    font-weight: 600;
+    text-transform: uppercase;
+    letter-spacing: 0.025em;
+    background-color: #6772e5;
+    text-decoration: none;
+    -webkit-transition: all 150ms ease;
+    transition: all 150ms ease;
+    margin-top: 10px;
+    width: 100%;
+  }
+  
+  button:hover {
+    color: #fff;
+    cursor: pointer;
+    background-color: #7795f8;
```

Comment:
Codacy found an issue: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5864152)